as2_msgs/CMakeLists.txt

```diff
@@ -45,6 +46,8 @@ set(MSGS_FILES
   "msg/BehaviorStatus.msg"
   "msg/YawMode.msg"
   "msg/GimbalControl.msg"
+  "msg/Geozone.msg"
+  "msg/Polygonlist.msg"
```

Comment:
Why not PolygonList? Messages names should be CamelCase, right?

Reply:
right, changed now

Reply:
I would treat Geozone as one word (compound word)